README.md

```diff
@@ -1,12 +1,41 @@
-# Next.js Blog Theme
+# Next.js Blog Theme Designed by [Bejamas](https://bejamas.io)
 
-A simple blog starter based on Next.js, Tailwind 3.0 and MDX support. Includes modern design with dark and light theme.
+[![Deply to Netlify Button](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/netlify-templates/nextjs-blog-theme)
 
-[Wizard](https://nextjs-wizard.netlify.app/) - create your own blog in few clicks and deploy on Netlify.
+A customizable blog starter using:
+- [Next.js](https://github.com/vercel/next.js) v12
+- [Tailwind](https://tailwindcss.com/) v3.0
+- Built-in [MDX](https://mdxjs.com/) v1 support
+- Includes modern design with dark & light themes
 
-## Configure the blog
+![Preview of blog theme. Author named Jay Doe and blog's name is "Next.js Blog Theme" with one blog post](nextjs-blog-theme-preview.png)
 
-The config is based on ENV Variables to make it easy to integrate with any Jamstack platform like Netlify.
+[Take a gander at the demo.](https://bejamas-nextjs-blog.netlify.app)
+
+### Getting Started
+
+You can get started with this project in two ways: locally or using the [setup wizard](https://nextjs-wizard.netlify.app/).
+
+#### Setting Up Locally
+
+If you're doing it locally, start with clicking the [use this template](https://github.com/netlify-templates/nextjs-blog-theme/generate) button and clone the repository. Then after navigating in the terminal to the project, you can run these commands:
```

Comment:
Small suggestion: I'd explain what the `use this template` button does for people who might not be familiar with it. Something like "use this template button that will create a new repo with this template's files on your GitHub account. Once this is done, clone your new repository and navigate to it in your terminal".

Otherwise there could be some misunderstanding when reading "clone this repository", some people might understand clone the template repo not the new one created on their account.